src/Xamarin.Legacy.Sdk/Sdk/AutoImport.Android.props

```diff
@@ -23,7 +23,7 @@ NOTE! everything must be conditioned behind:
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\font\*.ttc" />
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\raw\*" Exclude="$(MonoAndroidResourcePrefix)\raw\.*" />
```

Comment:
Can we make the same checks for `$(MonoAndroidResourcePrefix)`?

If there are several lines, we can put them in an `<ItemGroup>` block with a single `Condition`.

Reply:
> Can we make the same checks for `$(MonoAndroidResourcePrefix)`?

Sure we can. I could not find other prop in the chat.

> If there are several lines, we can put them in an `<ItemGroup>` block with a single `Condition`.

Changed. Both `MonoAndroidResourcePrefix` and `MonoAndroidAssetPrefix` checks are now in separate `ItemGroup`